lib/helper.js

```diff
@@ -0,0 +1,28 @@
+var semver = require('semver');
+var normalizeGitUrl = require('normalize-git-url');
+
+exports.DEFAULT_LB_VERSION = '2.x';
+exports.MASTER_LB_VERSION = '3.x';
+
+/**
+ * parse loopback version
+ * @param version
+ * @returns parse loopback version or default if invalid
+ */
+exports.parseLoopBackVersion = function(version) {
+  // If its a valid loopback version, return it
+  var validRange = semver.validRange(version);
+  if (validRange != null) {
+    return version;
+  }
+  // Parse git url
+  var normalized = normalizeGitUrl(version);
+  var branch = normalized.branch;
+  // If its the master branch, return the master branch version.
+  if ((/master/i).test(branch)) {
```

Comment:
This will match any branch name containing `master`, case-insensitive. Is that intentional?

I would prefer to use a simpler condition `if (branch === 'master')`, what is the benefit of using a regular expression?